app/src/main/java/se/inera/intyg/minaintyg/certificate/service/dto/GetCertificateRequest.java

```diff
@@ -0,0 +1,15 @@
+package se.inera.intyg.minaintyg.certificate.service.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
```

Comment:
This can be @Value and @Builder as it is not used for serialisation/deserialisation.